addons/web/static/src/js/views/calendar/calendar_view.js

```diff
@@ -195,7 +195,7 @@ var CalendarView = AbstractView.extend({
         this.loadParams.filters = filters;
         this.loadParams.mode = (params.context && params.context.default_mode) || attrs.mode;
         this.loadParams.scales = scales;
-        this.loadParams.initialDate = moment(params.initialDate || new Date());
+        this.loadParams.initialDate = moment((params.context && params.context.initialDate) || params.initialDate || new Date());
```

Comment:
Could be great to add a test ( :D ) + a doc with a small example somewhere (at least in commit message) ?